app/views/docs/authentication-management.phtml

```diff
@@ -0,0 +1,192 @@
+<p>
+    Appwrite has built-in features to help manage user accounts.
+    Users can be organized into teams and be given labels, so they can be given different permissions and access different resources.
+    Each user can also have their own preference object, which you can use to save preferences such as theme, language, and notification settings.
+</p>
+
+[TODO @steven: TEAMS docs]
```

Comment:
Teams are a good way to allow users to collaborate and share access to resources. For example, in a todo app, a user can [create a team](https://appwrite.io/docs/client/teams#teamsCreate) for one of their todo lists and [invite another user](https://appwrite.io/docs/client/teams#teamsCreateMembership) to the team to grant the other user access. The invited user can [accept the invitation](https://appwrite.io/docs/client/teams#teamsUpdateMembershipStatus) to gain access or ignore it if they don't want to gain access.

If you'd like for users to collaborate without requiring users to accept the invitation, use an Appwrite Function and server SDK to [add a user to a team](https://appwrite.io/docs/server/teams#teamsCreateMembership).